app/components/Sidebar/components/CollectionLink.tsx

```diff
@@ -197,19 +199,21 @@ function CollectionLink({
   const isDraggingAnyCollection =
     isDraggingAnotherCollection || isCollectionDragging;
 
+  React.useEffect(() => {
+    if (isDraggingAnyCollection) {
+      setExpanded(false);
```

Comment:
this will close any open collection because of a drag without re-opening. collections that were previously open should become open again once the drag finishes